src/main/resources/db/migration/changes/ZZ-LastChanges.json

```diff
@@ -0,0 +1,94 @@
+{
+    "databaseChangeLog": [
+        {
+            "changeSet": {
+              "id": "ZZ-1",
+              "author": "matt (generated)",
+              "changes": [
+                {
+                  "createIndex": {
+                    "columns": [
+                      {
+                        "column": {
+                          "name": "USER_GITHUB_ID"
+                        }
+                      }]
+                    ,
+                    "indexName": "FK8WOD0WRCEOIFBBPWONE12SMEV_INDEX_9",
+                    "tableName": "USEREMAILS"
+                  }
+                }]
+              
+            }
+          },
+          
+          {
+            "changeSet": {
+              "id": "ZZ-2",
+              "author": "matt (generated)",
+              "changes": [
+                {
+                  "createIndex": {
+                    "columns": [
+                      {
+                        "column": {
+                          "name": "CREATED_BY_ID"
+                        }
+                      }]
+                    ,
+                    "indexName": "FKKJPYGUUYD5SHXTABV9V5JPE6X_INDEX_2",
```

Comment:
Where does this name come from?   Is it generate automatically?   Can we choose a naming convention that has the same effect?